data/hai/hai culture conversations.txt

```diff
@@ -207,3 +207,47 @@ mission "Gambling Judgment"
 			label end
 			`	He takes your order diligently and with a smile.`
 				decline
+
+
+mission "Robotic Musical 1"
+	minor
+	source
+		government "Hai"
+		attributes "tourism"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`Passing down the main thoroughfare of this spaceport you almost ignore a stack of haphazardly piled crates in the median strip. It's not uncommon for recently unloaded crates to temporarily sit in front of different businesses in a port, place conspicuously out of the way of the flow of foot traffic, but it is the disordered piling that seems out of place.`
+			`	No sooner do you slow your step to give it your attention though, than the pile of crates immediately spring to life.`
```

Comment:
```suggestion
			`	No sooner do you slow your step to give it your attention though that the pile of crates immediately springs to life.`
```

Reply:
It would definitely be inappropriate to use "that" in this structure in Australian English. Is this some Americanism?

Reply:
Original seems correct to me as well.

Reply:
Huh. Guess I just haven't seen enough of that `No sooner` phasing, but Google agrees with you on that. I will stand by the `pile of crates` being a collective noun in this instance though (see convo on line 226).
```suggestion
			`	No sooner do you slow your step to give it your attention though, than the pile of crates immediately springs to life.`
```